test/test.js

```diff
@@ -35,6 +35,36 @@ const placeholder = {
 };
 
 describe('RNPickerSelect', () => {
+    describe('when provided an itemKey prop', () => {
+        it('sets the selected item via key rather than value', () => {
+            const duplicateValueItems = [
```

Comment:
Do `duplicate values` have any relevance here to what the method is doing? It seems that the method will look look at the itemKey first if provided, so it doesn't ever look at the value so it doesn't matter if the values are duplicated.

Reply:
Right, good point – I named it that way because of behavior I encountered when items had different labels, different keys, but duplicate values. But perhaps that name isn't relevant in the tests.